config/routes.rb

```diff
@@ -8,6 +8,6 @@
   get 'logout', to: 'sessions#destroy', as: 'logout'
   get 'auth/:provider/callback', to: 'sessions#create'
   get 'auth/failure', to: redirect('/')
-
-  get 'repositories', to: 'repo#index'
+  put 'repositories/:id/update', to: 'repo#update'
```

Comment:
Use the resources method to define CRUD routes. 
https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions